src/main/java/com/trainignapp/trainingapp/model/Trainee.java

```diff
@@ -1,61 +1,16 @@
 package com.trainignapp.trainingapp.model;
 
+import jakarta.persistence.*;
+import lombok.Data;
 
-import java.time.LocalDate;
-
+import java.util.Date;
 
+@Entity
+@Data
```

Comment:
I would put @Entity over the class declaration. Technically there's no difference, but more readable that this class is @Entity

Reply:
Read how to use Lombok @Data if you use inheritance. 
you need to add @EqualsAndHashCode(callSuper = true)